app/modules/enduser_setup.c

```diff
@@ -626,6 +626,52 @@ static void do_station_cfg (task_param_t param, uint8_t prio)
 }
 
 
+/**
+ *
+ *
+ *
+ *
+ */
+static int enduser_setup_write_file_with_extra_configuration_data(
+char * configuration_string, int configuration_length
+)
+{
+  int p_file = 0;
+  lua_State *L = NULL;
+	
+  ENDUSER_SETUP_DEBUG("enduser: opening enduser_custom_parameters.json for write");
+
+  // setup the file output
+  p_file = vfs_open("enduser_custom_parameters.json", "w");
```

Comment:
May I suggest a shorted filename? Perhaps "eus_custom.json"? Feels like a lot of characters to type otherwise.
 
Actually, is it guaranteed to even be JSON? If not, maybe "eus_custom.txt"?